skytap/network.go

```diff
@@ -62,7 +62,7 @@ type VpnAttachmentNetwork struct {
 	ConfigurationID *string `json:"configuration_id"`
 }
 
-// VPN described a virtual machine attached to an environment.
+// VPN described a virtual machine attached to a network.
```

Comment:
The comment still not making sense, VPN attached to a network?

Reply:
done